content/docs/03-ai-sdk-core/15-tools-and-tool-calling.mdx

```diff
@@ -60,7 +59,17 @@ When `maxSteps` is set to a number greater than 1, the language model will be ca
 in a loop when there are tool calls and for every tool call there is a tool result, until there
 are no further tool calls or the maximum number of tool steps is reached.
 
-### Example: generateText
+### Example
+
+In the following example, there are two steps:
+
+1. **Step 1**
+   1. The prompt `'What is the weather in San Francisco?'` is sent to the model.
+   1. The model generates a tool call.
+   1. The tool call is executed.
+1. **Step 2**
+   1. The tool result is sent to the model.
```

Comment:
this is a smol nit but I wonder if this tool result being sent back to the model should be before step 2?

Reply:
hm i feel it's part of step 2 (def not step 1 and i don't want to overcomplicate things)